src/three/plugins/ImplicitTilingPlugin.js

```diff
@@ -0,0 +1,63 @@
+import {SUBTREELoader} from "./SUBTREELoader.js";
+
+export class ImplicitTilingPlugin {
+
+	constructor() {
+
+		this.name = 'IMPLICIT_TILING_PLUGIN';
+
+	}
+
+	init(tiles) {
+
+		this.tiles = tiles;
+
+	}
+
+	preprocessNode(tile, uri, parentTile) {
+
+		if (tile.implicitTiling) {	//only for root of the tileset
+
+			tile.__hasUnrenderableContent = true;
+			tile.__hasRenderableContent = false;
+
+			// Store the infos from the tileset
+			tile.__subtreeDivider =  tile.implicitTiling.subdivisionScheme === "QUADTREE" ? 4 : 8;
+			tile.__subtreeUri = tile.implicitTiling.subtrees.uri;
+
+			// Original content uri
+			tile.__contentUri = uri ?? tile.content?.uri;
+
+			// Declare some properties
+			tile.__subtreeIdx = 0;	// Idx of the tile in its subtree
+
+			// Coords of the tile
+			tile.__x = 0;
+			tile.__y = 0;
+			tile.__z = 0;
+
+			let implicitUri = tile.__subtreeUri.replace("{level}", (tile.__depth ?? tile.__level) ?? 0);
+			implicitUri = implicitUri.replace("{x}", "0");
+			implicitUri = implicitUri.replace("{y}", "0");
+			implicitUri = implicitUri.replace("{z}", "0");
+			tile.content.uri = new URL(implicitUri, tile.__basePath + '/').toString();
+		// Handling content uri pointing to a subtree file
+		}else if (/.subtree$/i.test(tile.content?.uri)){
+			tile.__hasUnrenderableContent = true;
+			tile.__hasRenderableContent = false;
+		}
+
+	}
+
+	parseTile(buffer, parseTile, extension) {
+
+		//todo use extension instead ?
+		if (/.subtree$/i.test(parseTile.content.uri)) {
+			const loader = new SUBTREELoader(parseTile, this.tiles.root);
```

Comment:
Given https://github.com/NASA-AMMOS/3DTilesRendererJS/pull/650#discussion_r1710874576 it doesn't seem correct to pass the tile set root tile into this object. Rather this should be the implicit tiling root that expanded this tile.

Reply:
Absolutely.
Then I need to find a way to keep this this expanding tile.
As I've commented on the previous comment, I didn't test this case :/

Reply:
This should be a minor change, I think - see https://github.com/NASA-AMMOS/3DTilesRendererJS/pull/650#discussion_r1713001941.